### PR TITLE
Fix 2067: Comments in anonymous record values are deleted

### PR DESCRIPTION
**Pull request by ribeirotomas1904**
Fixes: #2067

### CODE REVIEW

File: src/Fantomas.Tests/RecordTests.fs [hunk 1]
```diff
@@ -1894,3 +1894,27 @@ let parse (checker: FSharpChecker) (parsingOptions: FSharpParsingOptions) { File
...
+    BarValue = barValue
+|}
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+{|
+// The foo value.
```
Comment (ribeirotomas1904) [inline src/Fantomas.Tests/RecordTests.fs:1916]: I'm having an unexpected output regarding the styling, I would guess that I have to change something around [here](https://github.com/fsprojects/fantomas/blob/master/src/Fantomas/CodePrinter.fs#L3053), can you confirm if that's the case? Also, what would be the expected styling/output in this case?

**Comment by ribeirotomas1904** [comment]
When possible, please give me some feedback regarding naming (test, variables), as well as any other change necessary to make the PR acceptable

PS: Thanks for all the effort to make it easier for newcomers to contribute

File: src/Fantomas.Tests/RecordTests.fs (hunk 1 above)
Comment (nojaf) [reply src/Fantomas.Tests/RecordTests.fs:1916]: Yes, I was expecting it more to be something like
```fsharp
{| // The foo value.
   FooValue = fooValue
   // The bar value.
   BarValue = barValue |}
```
I'll need to look into why this is happening, I'll get back to you.

File: src/Fantomas.Tests/RecordTests.fs [hunk 2]
```diff
@@ -1894,3 +1894,27 @@ let parse (checker: FSharpChecker) (parsingOptions: FSharpParsingOptions) { File
         })
     |> Async.Parallel
 """
+
+[<Test>]
+let ``anonymous records with comments on members, 2067`` () =
```
Comment (nojaf) [inline src/Fantomas.Tests/RecordTests.fs:1899]: Maybe change to `with comments on record fields`, as they are not really members.

**Comment by nojaf** [comment]
Hey, you are off to a good start here! Thanks for this PR.
I'll need to look into why the result is a bit weird.

File: src/Fantomas.Tests/RecordTests.fs (hunk 2 above)
Comment (ribeirotomas1904) [reply src/Fantomas.Tests/RecordTests.fs]: Fixed on b2550b46813c5f7daece9245b1468abbee7f3307

**Comment by nojaf** [comment]
Could you add this test to `MultilineBlockBracketsOnSameColumnRecordTests.fs`
```fsharp
[<Test>]
let ``anonymous records with comments on record fields`` () =
    formatSourceString
        false
        """
{|
    // The foo value.
    FooValue = fooValue
    // The bar value.
    BarValue = barValue
|}
"""
        config
    |> prepend newline
    |> should
        equal
        """
{|
    // The foo value.
    FooValue = fooValue
    // The bar value.
    BarValue = barValue
|}
"""
```
This is more likely the behaviour the user is looking for and you also fixed this in this PR.

**Comment by ribeirotomas1904** [comment]
> Could you add this test to `MultilineBlockBracketsOnSameColumnRecordTests.fs`
> 
> This is more likely the behaviour the user is looking for and you also fixed this in this PR.

Yes, tonight I will take some time to add this and fix the regular one

**Review by nojaf — APPROVED** [review]
Many thanks for this PR @ribeirotomas1904

**Pull request merged**